orderer/consensus/etcdraft/util.go

```diff
@@ -196,6 +196,48 @@ func ConsensusMetadataFromConfigBlock(block *common.Block) (*etcdraft.ConfigMeta
 	return MetadataFromConfigUpdate(configUpdate)
 }
 
+// ConsensusMetadataFromConfigBlockConfig reads consensus metadata from the configuration block's Config
+func ConsensusMetadataFromConfigBlockConfig(block *common.Block) (*etcdraft.ConfigMetadata, error) {
+	if block == nil {
```

Comment:
This name is funny... `ConsensusMetadataFromConfigBlock` is only called in one place to I guess it won't be too much to rename it to `ConsensusMetadataFromConfigUpdateBlock` and use that name here?

Or maybe something like `ExtractConsensusTypeMetadata` ?

Reply:
Went with the first suggestion